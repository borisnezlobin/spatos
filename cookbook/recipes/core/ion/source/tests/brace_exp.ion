echo '# ANCHOR: single_brace_expansion'
echo filename.{ext1,ext2}
echo '# ANCHOR_END: single_brace_expansion'
echo '# ANCHOR: multi_brace_expansion'
echo job_{01,02}.{ext1,ext2}
echo '# ANCHOR_END: multi_brace_expansion'
echo '# ANCHOR: nested_brace_expansion'
echo job_{01_{out,err},02_{out,err}}.txt
echo '# ANCHOR_END: nested_brace_expansion'
echo '# ANCHOR: range_brace_expansion'
echo {1..10}
echo {10..1}
echo {1...10}
echo {10...1}
echo {a..d}
echo {d..a}
echo {a...d}
echo {d...a}
echo '# ANCHOR_END: range_brace_expansion'
echo '# ANCHOR: range_brace_expansion_as_array'
for num in {1..10}
    echo $num
end
echo '# ANCHOR_END: range_brace_expansion_as_array'
