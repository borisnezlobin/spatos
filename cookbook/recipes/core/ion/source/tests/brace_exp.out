# ANCHOR: single_brace_expansion
filename.ext1 filename.ext2
# ANCHOR_END: single_brace_expansion
# ANCHOR: multi_brace_expansion
job_01.ext1 job_01.ext2 job_02.ext1 job_02.ext2
# ANCHOR_END: multi_brace_expansion
# ANCHOR: nested_brace_expansion
job_01_out.txt job_01_err.txt job_02_out.txt job_02_err.txt
# ANCHOR_END: nested_brace_expansion
# ANCHOR: range_brace_expansion
1 2 3 4 5 6 7 8 9
10 9 8 7 6 5 4 3 2
1 2 3 4 5 6 7 8 9 10
10 9 8 7 6 5 4 3 2 1
a b c
d c b
a b c d
d c b a
# ANCHOR_END: range_brace_expansion
# ANCHOR: range_brace_expansion_as_array
1
2
3
4
5
6
7
8
9
# ANCHOR_END: range_brace_expansion_as_array
